src/CAREUI/display/ScrollableColumn.tsx

```diff
@@ -0,0 +1,28 @@
+// src/CAREUI/display/ScrollableColumn.tsx
+import React from "react";
+
+interface ScrollableColumnProps {
+  title: string;
+  children: React.ReactNode;
+}
+
+const ScrollableColumn: React.FC<ScrollableColumnProps> = ({
+  title,
+  children,
+}) => {
+  return (
+    <div
+      style={{
+        height: "400px",
+        overflowY: "auto",
+        border: "1px solid #ccc",
+        margin: "10px",
+      }}
+    >
```

Comment:
1. Let's use shadcn's ScrollArea instead of custom building it https://ui.shadcn.com/docs/components/scroll-area

2. Since this is a tailwindcss based project, we do not use css styles if tailwind classes can do it.